src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+.card {
+  width: $cardWidth;
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: auto;
```

Comment:
```suggestion
    margin: 0 auto;
```

that'll happen if the card become higher 🥲

<img width="218" alt="image" src="https://user-images.githubusercontent.com/55802630/173081650-23860443-07eb-424a-b215-ca59f4bb4818.png">